homework/index.js

```diff
@@ -30,14 +36,102 @@
     return elem;
   }
 
+  function createOptions(repoNames, select) {
+    createAndAppend('option', select, { text: 'Select module', disabled: 'disabled' });
+    repoNames.sort((a, b) => a.name.localeCompare(b.name));
+    repoNames.forEach((repository, i) => {
+      createAndAppend('option', select, {
+        text: repository.name,
+        value: repository.name,
+      });
+    });
+  }
+
+  function renderRepoInformation(repo, ul) {
+    const url = `https://api.github.com/repos/HackYourFuture/${repo}`;
+    fetchJSON(url, (err, repoInfos) => {
+      if (err) {
+        renderError(err);
+        return;
+      }
+
+      ul.innerHTML = ' ';
+      const li = createAndAppend('li', ul);
+      const table = createAndAppend('table', li, { class: 'infoTable' });
+      const tbody = createAndAppend('tbody', table);
+      const repoName = createAndAppend('tr', tbody);
+      createAndAppend('td', repoName, { text: 'Repository:', class: 'explanations' });
+      const repoNameTd = createAndAppend('td', repoName, {
+        class: 'list-item',
+      });
+      createAndAppend('a', repoNameTd, {
+        href: `${repoInfos.html_url}`,
```

Comment:
There is no need to use template literals here if they only contain something that is a string already (or is converted to string automatically):

```js
href: repoInfos.html_url,
````

There are several places in your code where you use unneeded template literals.